pinot-broker/src/main/java/org/apache/pinot/broker/routing/RoutingTable.java

```diff
@@ -20,10 +20,11 @@
 
 import java.util.List;
 import java.util.Map;
+import org.apache.pinot.core.routing.RouteTable;
 import org.apache.pinot.core.transport.ServerInstance;
 
 
-public class RoutingTable {
+public class RoutingTable implements RouteTable {
```

Comment:
Name the interface `RoutingTable` and rename the implementation to `RoutingTableImpl`